components/ui-api-layer/Jenkinsfile

```diff
@@ -44,14 +44,6 @@ podTemplate(label: label) {
                             execute("./before-commit.sh ci")
                         }
 
-                        if (params.FULL_BUILD) {
```

Comment:
if you removed from ui-api-layer, you can also remove from ui-api-layer-acc-tests